elyra/tests/metadata/test_utils.py

```diff
@@ -193,7 +193,7 @@ def create_instance(metadata_store: MetadataStore, location: str, name: str, con
 
 
 def get_unfiltered_schema(schema_name):
-    schema_file = os.path.join(os.path.dirname(__file__), '..', 'schemas', schema_name + '.json')
+    schema_file = os.path.join(os.path.dirname(__file__), '../../metadata', 'schemas', schema_name + '.json')
```

Comment:
I think this might be problematic on Windows and introduce mixed slashes.
```suggestion
    schema_file = os.path.join(os.path.dirname(__file__), '..', '..', 'metadata', 'schemas', schema_name + '.json')
```
For example, this on Windows...
```
>>> os.path.join(os.getcwd(), '../../tests', 'resources', 'components')
'C:\\Users\\Administrator\\notebooks\\../../tests\\resources\\components'
```
Another option is to wrap the join with unix slashes in `os.path.realpath()`:
```
>>> os.path.realpath(os.path.join(os.getcwd(), '../../tests', 'resources', 'components'))
'C:\\Users\\tests\\resources\\components'
```